include/re_sipsess.h

```diff
@@ -7,6 +7,15 @@
 struct sipsess_sock;
 struct sipsess;
 
+/* SDP Negotiation state */
+enum sdp_neg_state {
+	SDP_NEG_NONE = 0,
+	SDP_NEG_LOCAL_OFFER = (1 << 0),		/** SDP offer sent */
+	SDP_NEG_REMOTE_OFFER = (1 << 1),	/** SDP offer received */
+	SDP_NEG_PREVIEW_ANSWER = (1 << 2),	/** SDP preview answer sent */
+	SDP_NEG_DONE = (1 << 3)			/** SDP negotiation done */
+};
```

Comment:
is this a "state" or "flags" ?

If it is flags then the stored value can contain a bitmask of many values ORed together.

If it is a state, the value can only contain one value and the states values `1<<N` is not needed.

Reply:
Initially I thought it could be useful in some cases to OR the values (like flags), but I didn't end up using it anywhere. I have removed the explicit enum values.